### ISSUE TITLE
[TW-1635] Running "task anystringatall" does not filter anything

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1659]
_Tomas Babej on 2015-07-16T12:44:24Z says:_ 

 In 2.4.4, any string in "task <string>" used to be interpreted as implicit filter for description.
In 2.4.5, it seems that such a string is simply ignored.

```
$ task anystringatall
... lots of tasks
188 tasks, truncated to 61 lines
```

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2015-07-16T12:44:24Z
Modified: 2015-07-19T19:28:26Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2015-07-19T13:34:16Z says:_ 

 I cannot reproduce this:

```
▸ cat pending.data 
[description:"onetwothree" entry:"1437312978" modified:"1437312978" status:"pending" uuid:"77f530e4-2090-4c5d-867a-4384787249a1"]
[description:"twothreefour" entry:"1437312978" modified:"1437312978" status:"pending" uuid:"6612a9d0-a24b-4357-85b0-5118b35be806"]

▸ task asdf
No matches.

▸ task on

ID Age Description Urg 
 1 32s onetwothree    0

1 task

▸ task tw

ID Age Description  Urg 
 1 35s onetwothree     0
 2 35s twothreefour    0

2 tasks

▸ task fo

ID Age Description  Urg 
 2 38s twothreefour    0

1 task
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2015-07-19T19:13:22Z says:_ 

 Either cannot reproduce, or inadvertently fixed.

**Comment by taskwarrior** [comment]
_Tomas Babej on 2015-07-19T19:28:26Z says:_ 

 Confirmed, the issue was fixed with the commit 6aaa7c5074f61aaabb8629b0a3e8a0d26ba984ba. Which was, by the way, introduced 8 hours after I reported this.
